### PR TITLE
Move from string types to PHPStan types

### PR DESCRIPTION
**Pull request by TomasVotruba**
This will merge writing rules for both Rector and PHPStan to 1 skill :)

### CODE REVIEW

File: packages/DeadCode/src/Rector/Class_/RemoveSetterOnlyPropertyAndMethodCallRector.php [hunk 1]
```diff
@@ -107,22 +108,28 @@ public function refactor(Node $node): ?Node
 
     /**
      * @param Property|ClassMethod|MethodCall|Assign $node
-     * @return string[][]][]|null
+     * @return string[][] ][]|null
```
Comment (OskarStark) [inline packages/DeadCode/src/Rector/Class_/RemoveSetterOnlyPropertyAndMethodCallRector.php]: Hmm 🤔

File: packages/CodeQuality/src/Rector/FuncCall/SimplifyStrposLowerRector.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ public function getNodeTypes(): array
     }
 
     /**
-     * @param FuncCall $node;
+     * @param FuncCall $node ;
```
Comment (OskarStark) [inline packages/CodeQuality/src/Rector/FuncCall/SimplifyStrposLowerRector.php]: Is this correct?

Comment (TomasVotruba) [reply packages/CodeQuality/src/Rector/FuncCall/SimplifyStrposLowerRector.php]: I don't thinks so. PR?

File: packages/DeadCode/src/Rector/Class_/RemoveSetterOnlyPropertyAndMethodCallRector.php (hunk 1 above)
Comment (TomasVotruba) [reply packages/DeadCode/src/Rector/Class_/RemoveSetterOnlyPropertyAndMethodCallRector.php]: PR?

**Pull request merged**